### PR TITLE
Update css-color-function to enable `contrast()`

### PR DESCRIPTION
**Pull request by vanwagonet**

### CODE REVIEW

**Comment by vanwagonet** [comment]
@MoOx I'm hoping to get `contrast()` piped all the way into cssnext. Can you take a look?

**Comment by MoOx** [comment]
^1.1.1 is compatible with 1.2.0, so uninstall and reinstalling the package and it should be good. You can also delete the package in your npm cache to be sure to get it.

That being said, I will probably merge tomorrow morning.

File: test/fixtures/color.css [hunk 1]
```diff
@@ -2,4 +2,5 @@ body {
   color: color(red);
   background-color: color(red tint(50%));
   border-color: color(hsla(125, 50%, 50%, .4) hue(200));
+  border-top-color: color(hwb(270, 10%, 0%) contrast(50%));
```
Comment (MoOx) [inline test/fixtures/color.css:5]: We don't need to test what is already tested in another module :)

**Comment by MoOx** [comment]
Released as 1.2.0

**Pull request merged**